.github/workflows/pr.yaml

```diff
@@ -0,0 +1,39 @@
+on: pull_request
+name: PR Review
+jobs:
+  test-composer-files:
+    name: Validate composer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.2
+          extensions: ctype, dom, iconv, json, zip, gd, soap
+          coverage: none
+          tools: composer:v2
+      - name: Validate composer files
+        run: |
+          composer validate composer.json
+
+  phpcs:
+    name: PHP - Check Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.2
+          extensions: ctype, dom, iconv, json, zip, gd, soap
+          coverage: none
+          tools: composer:v2
+      - name: Install Dependencies
+        run: |
+          composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
```

Comment:
I typically choose to NOT have the `-q` and `-no-progress` flags to be able to easily spot problems on install. 

This is only test dependencies so possibly less important here. But remember that we are also implicitly testing if `composer.lock` is installable (`validate` doesn´t do this)

Reply:
I got this from another open source project: https://github.com/itk-dev/os2loop/blob/develop/.github/workflows/pr.yaml#L34

Reply:
We have been doing both and there are good arguments both for and against `-q`. The argument for `-q` is that it the output is 'noise' and a waste of bits/storage to have the extra output on all runs. The argument against is the one stated. That you get no usable output in case of failure. 

So it depends what your action is testing. You can argue that you should have a dedicated step to test the install without `-q`, and then do all other steps with `-q`